package.json

```diff
@@ -205,7 +205,7 @@
     "trunc-html": "1.0.2",
     "trunc-text": "1.0.2",
     "type-detect": "^4.0.8",
-    "uglifyjs-webpack-plugin": "0.4.6",
+    "uglifyjs-webpack-plugin": "1.2.7",
```

Comment:
nit: can we ^?  this missing ^ predates our yarn.lock file

Reply:
thanks for the review @jbudz changed 😄